db_test.go

```diff
@@ -652,6 +653,29 @@ func TestInCriteria(t *testing.T) {
 	})
 }
 
+func TestContainsCriteria(t *testing.T) {
+	runCloverTest(t, containsPath, func(t *testing.T, db *c.DB) {
+		docs, err := db.Query("contains").Where(c.Field("myField").Contains(4)).FindAll()
+		require.NoError(t, err)
+
+		require.Equal(t, 2, len(docs))
+
+	OuterLoop:
```

Comment:
This label is not needed

Reply:
The test will fail without the label; it is used to break out of the inner loop early, as soon as the test element is found. If the inner loop is not broken out of, it will proceed to the `Fail` function.

Reply:
There are several ways to write this without using the label. Using full jumps is not a good practice, not only in golang.
For example you could write:

```go
found := false
for _, elem := range myField {
       if elem.(float64) == 4 {
	      found = true
              break
	}
}
require.True(t, found)
```